app/graphql/types/user_type.rb

```diff
@@ -14,6 +14,9 @@ class UserType < Types::BaseObject
     field :monthly_moods, [Types::MoodType], null: true do
       argument :month, Int, required: false
     end
+    field :monthly_habits, [Types::HabitEntryType], null: true do
```

Comment:
I'm not sure what the null: true does. Is it needed for this query?

Reply:
I allows the field to return nil for a query I'm pretty sure. I'm not sure if we need that specifically for this query. If there are no habits completed for that month it should return an empty array